redfish-core/lib/pcie_slots.hpp

```diff
@@ -324,6 +373,8 @@ inline void getPCIeSlots(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                                             "Get", locationInterface,
                                             "LocationCode");
                                 }
+                                getLinkedPcieDevice(asyncResp, pcieSlotPath,
```

Comment:
A function with "get" in the name should return something shouldn't it?  addLinkedPCIeDevices?  (devices is plural here because the redfish spec (and the pcie spec) allows multiple devices in one slot ((it's fine if we only support a single device for the moment))).

Reply:
Updated.